MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Layout in .NET MAUI DataForm control | Syncfusion
+description: Learn here all about the Layout support in Syncfusion .NET MAUI DataForm (SfDataForm) control and more.
+platform: maui
+control: SfDataForm
+documentation: ug
+---
+
+# Layout in .NET MAUI DataForm (SfDataForm)
+
+## Overview
+
+The data form supports linear and grid layouts and manages layout of label, editor, and validation label.
+
+## Linear data form layout
+
+By default, the data form arranges the fields one-by-one. It is applicable for both label positions: left and top.
```

Comment:
By default, the data form arranges the fields one by one. It is applicable for both label positions: left and top.

Reply:
Content changed